src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -610,11 +628,7 @@ export default class CanvasController {
                 -round(UiController.km_to_px(fix.relativePosition[1])) + this.canvas.panY
```

Comment:
(_I cant comment on the line I want, this is high up as I can go_)

can we refactor the `_forEach` to a native `for`?